aodndata/moorings/products_handler.py

```diff
@@ -109,6 +110,7 @@ def __init__(self, *args, **kwargs):
         self.allowed_extensions = ['.json_manifest', '.nc', '.zip']
         self.product_site_code = None
         self.product_variables = None
+        self.products_to_create = VALID_PRODUCTS
```

Comment:
I would set this along FILE_INDEX_LAYER since it is not owned by HandlerBase.

Reply:
Done